apps/src/applab/designMode.js

```diff
@@ -250,6 +250,15 @@ designMode.updateProperty = function (element, name, value) {
       break;
     case 'icon-color':
       element.setAttribute('data-icon-color', value);
+      const imageUrl = element.getAttribute('data-canonical-image-url');
+      if (ICON_PREFIX_REGEX.test(imageUrl)) {
+        const url = assetPrefix.renderIconToString(imageUrl, element);
+        if (element.nodeName == "IMG") {
```

Comment:
Thanks Dave.  Sorry about that - I forgot that commits on Larry's fork don't trigger a Circle build.  Reverted in https://github.com/code-dot-org/code-dot-org/pull/13546 and I'll fix and unrevert tomorrow.